apps/pkt-gen/pkt-gen.c

```diff
@@ -1005,6 +1006,85 @@ update_addresses(struct pkt *pkt, struct targ *t)
 	else
 		update_ip6(pkt, t);
 }
+
+static void
+update_ip_size(struct pkt *pkt, struct targ *t, int size)
```

Comment:
The compiler whines about `*t` not being unused here and in `update_ip6_size()`.
Should it be removed or mark `__unused`?